packages/playwright-crawler/src/internals/utils/playwright-utils.ts

```diff
@@ -609,18 +609,22 @@ export async function parseWithCheerio(
 
         await Promise.all(
             frames.map(async (frame) => {
-                const iframe = await frame.contentFrame();
-
-                if (iframe) {
-                    const contents = await iframe.content();
-
-                    await frame.evaluate((f, c) => {
-                        const replacementNode = document.createElement('div');
-                        replacementNode.innerHTML = c;
-                        replacementNode.className = 'crawlee-iframe-replacement';
-
-                        f.replaceWith(replacementNode);
-                    }, contents);
+                try{
+                    const iframe = await frame.contentFrame();
+
+                    if (iframe) {
+                        const contents = await iframe.content();
+
+                        await frame.evaluate((f, c) => {
+                            const replacementNode = document.createElement('div');
+                            replacementNode.innerHTML = c;
+                            replacementNode.className = 'crawlee-iframe-replacement';
+
+                            f.replaceWith(replacementNode);
+                        }, contents);
+                    }
+                } catch(error) {
+                    console.warn('Failed to extract iframe content:', error);
```

Comment:
Use `log.warning` instead of `console.warn`.

Reply:
I have made the changes